src/main/scala/edu/ie3/simona/service/weather/WeatherService.scala

```diff
@@ -219,22 +209,22 @@ final case class WeatherService(
       longitude,
     )
 
+    val RegistrationMessage = serviceStateData.maybeNextActivationTick match {
```

Comment:
Val name that starts uppercase? 🤨 Also you could inlcude the `ctx.self.toClassic` here already, so you wouldn't need a val function. 
Or even better yet, include the whole `agentToBeRegistered ! RegistrationMessage(ctx.self.toClassic)` within the function?